certbot-compatibility-test/setup.py

```diff
@@ -31,11 +31,11 @@
 ]
 
 setup(
-    name='letsencrypt-compatibility-test',
+    name='certbot-compatibility-test',
     version=version,
-    description="Compatibility tests for Let's Encrypt client",
+    description="Compatibility tests for Certbot",
```

Comment:
I think we've agreed to have this be lowercase when not at the start of a sentence